exercises/D-logical-operators/exercise2.js

```diff
@@ -5,9 +5,26 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
-/* 
+function isNegative(num) {
+  if (num > 0) {
+    return false;
+  } else if (num < 0) {
+    return true;
+  }
+  return 0;
```

Comment:
Because this function is expected to return a boolean value (true or false), it's probably better not to return 0 in this case. The 0 could also return false here.